Casks/f/fujifilm-tether-app.rb

```diff
@@ -0,0 +1,31 @@
+cask "fujifilm-tether-app" do
+  version "1.30.0,vvr6ik27"
+  sha256 "613e43b3688b6a362a661a767c43b22db87fe574ba41f2cecb88e07c56154437"
+
+  url "https://dl.fujifilm-x.com/support/software/tether-app-mac#{version.csv.first.no_dots}-#{version.csv.second}/FUJIFILM_TetherApp_Mac#{version.csv.first.no_dots}.pkg"
+  name "FUJIFILM TETHER APP"
+  desc "For Fujifilm GFX/X series camera tether shooting"
+  homepage "https://fujifilm-x.com/en-us/support/download/software/tether-app/"
+
+  livecheck do
+    url :homepage
+    regex(%r{Mac\sVersion:\s(\d+(?:\.\d+)+).*href=.*?tether-app-mac\d+-([a-zA-Z0-9]+)/}im)
+    strategy :page_match do |page, regex|
+      match = page.match(regex)\
+      next if match.blank?
+
+      "#{match[1]},#{match[2]}" if match
```

Comment:
```suggestion
      match = page.match(regex)
      next if match.blank?

      "#{match[1]},#{match[2]}"
```

@p-linnane Sniped my edit haha